README.md

```diff
@@ -37,6 +46,18 @@ logger.Info("Failed to fetch URL.",
 )
 ```
 
+## Development Status: Release Candidate 1
+The current release is `v1.0.0-rc.2`. No further breaking changes are *planned*
```

Comment:
Another PR then I would say haha :) really this can just be a grep in the Makefile